js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -5,10 +5,10 @@ import core from './ui.grid_core.modules';
 import { focusAndSelectElement, getWidgetInstance } from './ui.grid_core.utils';
 import { isDefined } from '../../core/utils/type';
 import { inArray } from '../../core/utils/array';
-import { focused } from '../widget/selectors';
+import selectors from '../widget/selectors';
```

Comment:
Why is import specific selector replaced to import all selectors? Import specific method is better for tree shaking, is not it?